plutus-core/plutus-core/src/PlutusCore/Default/Builtins.hs

```diff
@@ -441,6 +446,7 @@ instance Flat DefaultFun where
               MkNilData                -> 50
               MkNilPairData            -> 51
               MkCons                   -> 52
+              Blake2b_256              -> 53
```

Comment:
This isn't relevant to this PR, but could we move `Nop1`, `Nop2`, and `Nop3` down to the bottom of this list and add a comment saying that any new builtins should be added before them?  They're only there for use in benchmarks so that we can get some idea of the overhead of calling a builtin.  They should never appear in real programs and I might be able to remove them at some point, so having them in the middle makes the encoding a bit unstable.

Reply:
True, but then again what kind of encoded tag number you are going to give to Nop1,Nop2,Nop3? For now, I just try to add builtins to the end of the list of builtins so as to not mess with the tags. I also rearranged the flat instance so the tags appear in order.

Reply:
Ok, I will turn them into `error "not supposed to serialise nop`, and move the remainder builtins more "up" in the tag order. I should not forget to regenerate the flats of validation + marlowe :(